src/components/filters/filter-container.component.html

```diff
@@ -1,21 +1,30 @@
 <ng-content></ng-content>
 
 <!-- Add a Clear Button -->
-<button type="button"
-    class="btn btn-link btn-icon btn-secondary m-l-xs"
-    tabindex="0"
-    aria-label="Clear all filters"
-    i18n-aria-label
-    *ngIf="(filterService.filters$ | async).length > 0"
-    [uxTooltip]="clearTooltip || 'Clear All'"
-    (click)="filterService.removeAll()">
+<ng-container *ngIf="!clearAll">
+    <button type="button"
+        class="btn btn-link btn-icon btn-secondary m-l-xs"
+        tabindex="0"
+        aria-label="Clear all filters"
+        i18n-aria-label
+        *ngIf="(filterService.filters$ | async).length > 0"
+        [uxTooltip]="clearTooltip || 'Clear All'"
+        (click)="filterService.removeAll()">
 
-    <svg class="filter-selected-clear-graphic" width="100%" viewBox="0 0 19 12" shape-rendering="geometricPrecision">
-        <rect class="light-grey" x="0" y="2" width="7" height="2"></rect>
-        <rect class="dark-grey" x="0" y="5" width="9" height="2"></rect>
-        <rect class="light-grey" x="0" y="8" width="7" height="2"></rect>
-        <path class="dark-grey" d="M9,1 h1 l9,9 v1 h-1 l-9,-9 v-1 Z"></path>
-        <path class="dark-grey" d="M9,11 v-1 l9,-9 h1 v1 l-9,9 h-1 Z"></path>
-    </svg>
+        <svg class="filter-selected-clear-graphic" width="100%" viewBox="0 0 19 12" shape-rendering="geometricPrecision">
+            <rect class="light-grey" x="0" y="2" width="7" height="2"></rect>
+            <rect class="dark-grey" x="0" y="5" width="9" height="2"></rect>
+            <rect class="light-grey" x="0" y="8" width="7" height="2"></rect>
+            <path class="dark-grey" d="M9,1 h1 l9,9 v1 h-1 l-9,-9 v-1 Z"></path>
+            <path class="dark-grey" d="M9,11 v-1 l9,-9 h1 v1 l-9,9 h-1 Z"></path>
+        </svg>
+    </button>
+</ng-container>
 
-</button>
\ No newline at end of file
+<div class="btn-group"
+    *ngIf="clearAll && (filterService.filters$ | async).length > 0">
+    <button type="button" tabindex="0" class="btn btn-link button-primary table-filter-clear"
+        (click)="filterService.removeAll()">
+        <ng-container *ngTemplateOutlet="clearAll"></ng-container>
+    </button>
```

Comment:
Usually the way we do this, to avoid duplicating code, is to provide the default content as a template, and then implement it as:
```html
    <ng-container
        [ngTemplateOutlet]="clearAllTemplate || defaultClearAllTemplate">
    </ng-container>
```
```html
<ng-template #defaultClearAllTemplate>
    <svg ...>
</ng-template>
```